encoding/base64/src/base64.c

```diff
@@ -167,6 +167,29 @@ base64_decode(const char *str, void *data)
     return q - (unsigned char *) data;
 }
 
+int
+base64_decode_maxlen(const char *str, void *data, int len)
```

Comment:
While it is visible that function body was copied from base64_decode() style could be refreshed since new lines do adhere to latest standards.